packages/contracts-bedrock/test/kontrol/scripts/run-kontrol.sh

```diff
@@ -105,13 +106,29 @@ regen=
 #################################
 # Tests to symbolically execute #
 #################################
-# Missing: OptimismPortalKontrol.prove_proveWithdrawalTransaction_paused
-test_list=( "OptimismPortalKontrol.prove_finalizeWithdrawalTransaction_paused" \
-            "L1StandardBridgeKontrol.prove_finalizeBridgeERC20_paused" \
-            "L1StandardBridgeKontrol.prove_finalizeBridgeETH_paused" \
-            "L1ERC721BridgeKontrol.prove_finalizeBridgeERC721_paused" \
-            "L1CrossDomainMessengerKontrol.prove_relayMessage_paused"
-          )
+
+test_list=()
+if [ "$SCRIPT_TESTS" == true ]; then
+  test_list=( # "OptimismPortalKontrol.prove_proveWithdrawalTransaction_paused0" \
+              # "OptimismPortalKontrol.prove_proveWithdrawalTransaction_paused1" \
+              "OptimismPortalKontrol.prove_proveWithdrawalTransaction_paused2" \
+              # "OptimismPortalKontrol.prove_proveWithdrawalTransaction_paused3" \
```

Comment:
I've removed the unexecuted lemmas out of the `test_list` so that the shellcheck error si not present anymore. Will add them back before merging this PR tho!